source/frontend/src/components/common/form/Select.tsx

```diff
@@ -100,10 +100,12 @@ export const Select: React.FC<React.PropsWithChildren<SelectProps>> = ({
   };
 
   const renderPlaceholder = () => {
-    if (!placeholder) {
+    const calculatedPlaceholder =
+      !value || options.find(option => option.value === value) ? null : 'N/A'; // Render N/A in the event that the currently selected value is not in the list.
```

Comment:
Note: this is required, otherwise if a value is currently selected that is not in the options list, the select will default to showing the first value in the options list, even if that is not what the select's current value is. To correct this, the user would have to select the value which currently appears selected, which is not intuitive.